### DISCUSSION TITLE
想交叉编译软件到3代笔上，但交叉编译出了问题

### DISCUSSION THREAD

**Discussion by xjackzenvey** [PenUniverse/PenMods#220] [你问我答]
在ubuntu22.04下用arm-linux-gnueabi-gcc编译PPSSPP(没错就是PSP模拟器)，但是提示找不到opengl库，然后opengl又编译不过。。。求解决方法。
编译环境：wsl ubuntu22.04 amd64
编译器：arm-linux-gnueabi (gcc 11.4)+Cmake
PPSSPP:https://github.com/hrydgard/ppsspp  (这玩意支持qt平台)

**Comment by Redbeanw44602** [comment]
有的时候并不需要自己编译，你可以找发行版Linux软件仓库里的编译好的，aarch64架构就行

**Reply by xjackzenvey** [reply]
发行版软件一上来就提示少各种库……有没有什么快捷的安装依赖的方法啊

**Reply by Redbeanw44602** [reply]
你可以给词典笔的embedded linux引入一个包管理器，不过其实我也没试过。缺少各种库不是因为它是发行版的软件，而是因为它的确需要哪些库，如果不用包管理器你只能手动解决依赖。

**Reply by xjackzenvey** [reply]
现在提示缺少glibc2.33 .......这个库怎么都装不上去了
